src/stories/Button/Button.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Button } from '@material-ui/core';
+import logout from '../../actions/logout';
```

Comment:
usless import

Reply:
So why eslint allows you to push it? You should add "no-useless-import" rule or something like that